test/integration-test/test_lib/test_analysis/test_user.py

```diff
@@ -50,5 +51,65 @@ def test_keywords(self):
         self.assertEqual(expected_sorted_keywords_for_channels, sorted_keywords_for_channels)
 
 
+    @patch("lib.analysis.user.time", autospec = True)
```

Comment:
Why are we patching a method in an integration test?

Reply:
@prasadtalasila In keywords_clusters function, we are printing the time taken for various operations (like feature extraction) to stdout. The time taken for such operations varies slightly each time the test is run. The time() function is patched to ensure that the running time printed to stdout is constant( zero in this case) every time the test is run.